models/Size.js

```diff
@@ -6,8 +6,12 @@ const SizeSchema = new mongoose.Schema({
   widthInCm: Number,
   depthInCm: Number,
   volumeInLiters: Number,
```

Comment:
I think we can store it `volume: String`. The actual value will look like "12liters" because I don't think that we are going to perform some calculations using volume and number will be useless here

Reply:
That would require applying the "language" structure (12liters / 12літрів). Also, is it "12liters" or "12 liters"? What if someone down the line wants to convert it after all? What are the benefits of going against 1NF like that?